test/testdrive/kafka-exactly-once-sink.td

```diff
@@ -58,11 +58,14 @@ $ kafka-create-topic topic=input
 
 > CREATE MATERIALIZED VIEW input_values_mview AS VALUES (1), (2), (3);
 
-> CREATE VIEW input_kafka_no_byo_join_view AS SELECT * FROM input_kafka_byo, input_kafka_no_byo;
```

Comment:
I think we should just get rid of these tests because they no longer test what was meant to be tested here (exactly once sinks can't depend on rt sources)

Reply:
I should delete the whole file?

Reply:
no just the tests where the error message changed to disallow selecting from multiple timelines